archunit/src/test/java/com/tngtech/archunit/core/importer/ClassFileImporterAnnotationsTest.java

```diff
@@ -554,6 +569,14 @@ public void parameters_of_meta_annotations_are_transitively_imported(HasAnnotati
         return (T) javaClass.getMethod(methodName).getDefaultValue().get();
     }
 
+    private @interface SimpleAnnotationWithDefaultValue {
+        String fallsBackToDefault() default "expected";
+
+        String hasDefaultButOverwritten() default "toOverwrite";
```

Comment:
I think I'd find it less confusing if the property names of the annotation wouldn't contain semantics of the test that is defined 470 lines away. (Unfortunately, the annotation cannot be a local class – but it could be defined right next to the test... 😜)

So I'd just use:
```java
String hasDefault1() default "default value 1";
String hasDefault2() default "default value 2";
String hasDefault2() default "default value 3";
```
instead of `fallsBackToDefault` and `hasDefaultButOverwritten`.

Reply:
Yeah, I know, I wish you could define it as local class :disappointed: I still like it to communicate what the test means :thinking: Maybe I can find a way that this part is only in the test.